test/export/InstanceExporter.test.ts

```diff
@@ -301,6 +325,26 @@ describe('InstanceExporter', () => {
       expect(exported.communication).toBeUndefined();
     });
 
+    it('should fix a nested element that has parents defined in the instance and fixed on the SD to an array even if constrained', () => {
+      const cardRule = new CardRule('communication');
```

Comment:
Should this `CardRule` apply to `contact` here? The test still passes when this change is made.

Reply:
Oops yes! Good catch!